### PR TITLE
Create the form page

### PR DESCRIPTION
**Pull request by Victoria129**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -163,7 +163,39 @@ <h4 class="poppins-regular">Skills<img src="https://img.icons8.com/small/16/0000
...
+      I’d love to help with it </p>
+
+    <div class="form-container">
+    <form action="https://formspree.io/f/xzbwropn" method="POST">
+
+      <ul class="form-list">
+
+        <li class="form-field">
+          <label for="name"></label>
+          <input class="text-field" type="text" name="name" id="name" max="30" placeholder="name" required>
```
Comment (jasonfelice) [inline index.html]: - [ ] Kindly note that the correct attribute is `maxlength="30"`. Please replace `max="30"`.

File: index.html [hunk 2]
```diff
@@ -163,7 +163,39 @@ <h4 class="poppins-regular">Skills<img src="https://img.icons8.com/small/16/0000
...
+        </li>
+
+        <li class="form-field">
+          <label for="email"></label>
+          <input class="text-field" type="email" name="email" id="email" placeholder="name@example.com" required>
+        </li>
+
+        <li class="form-field">
+          <label for="message"></label>
+          <textarea required id="message" max="500" name="message" placeholder="Write your message here"></textarea>
```
Comment (jasonfelice) [inline index.html]: - [ ] Kindly note that the correct attribute is `maxlength="500"`. Please replace `max="500"`.

**Review by jasonfelice — CHANGES_REQUESTED** [review]
Hi @Victoria129, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Good design✔️
Followed GitHub flow✔️
No linter errors✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Nyame-Wolf — APPROVED** [review]
# STATUS: APPROVED :heavy_check_mark: :green_circle: 

Hi @Victoria129  :smile: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :sparkles:

## Optional suggestions :ballot_box_with_check: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

APPROVED! 💯 💯 💯

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me, @Nyame-Wolf in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**